modules/ROOT/pages/apikit-graphql-api-implementation.adoc

```diff
@@ -0,0 +1,525 @@
+               "name": "Bonny Madgewick",
+               "email": "bmadgewick2g@mozilla.com",
+               "website": "utexas.edu",
+               "address": {
+                   "street": "5 David Terrace",
+                   "city": "Oelaba",
+                   "zipcode": null
+               }
+           },
+           ...
+           {
+               "name": "Gretal Titmuss",
+               "email": "gtitmuss2e@soup.io",
+               "website": "creativecommons.org",
+               "address": {
+                   "street": "3201 Gale Place",
+                   "city": "Bolo",
+                   "zipcode": "4201"
+               }
+           }
+       ]
+   }
+}
+----
+
+=== Retrieve a Post by ID
+
+This query example requests the post data by its ID, including the post comments and the usernames.
+
+==== Make a Query
+
+----
+{
+   getPost(id: 10) {
+       id
+       user {
+           username
+       }
+       title
+       body
+       comments {
+           user {
+               username
+           }
+           body
+           replies {
+               body
+           }
+       }
+   }
+}
+----
+
+==== Trigger a Request
+
+----
+curl --location --request POST 'http://localhost:8081/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{"query":"{getPost(id:10){id user{username}title body comments{user{username}body replies{body}}}}","variables":{}}'
+----
+
+==== Get a Response
+
+----
+{
+   "data": {
+       "getPost": {
+           "id": "10",
+           "user": {
+               "username": "bdearelli"
+           },
+           "title": "id luctus nec molestie sed justo pellentesque viverra pede ac",
+           "body": "Maecenas tristique, est et tempus semper, est quam pharetra magna, ac consequat metus sapien ut nunc. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Mauris viverra diam vitae quam. Suspendisse potenti.\n\nNullam porttitor lacus at turpis. Donec posuere metus vitae ipsum. Aliquam non mauris.",
+           "comments": [
+               {
+                   "user": {
+                       "username": "cwalesby1p"
+                   },
+                   "body": "Praesent blandit. Nam nulla. Integer pede justo, lacinia eget, tincidunt eget, tempus vel, pede.\n\nMorbi porttitor lorem id ligula. Suspendisse ornare consequat lectus. In est risus, auctor sed, tristique in, tempus sit amet, sem.",
+                   "replies": []
+               },
+               {
+                   "user": {
+                       "username": "rricci2d"
+                   },
+                   "body": "Maecenas leo odio, condimentum id, luctus nec, molestie sed, justo. Pellentesque viverra pede ac diam. Cras pellentesque volutpat dui.",
+                   "replies": []
+               }
+           ]
+       }
+   }
+}
+----
+
+== Request Mutations Examples
+
+The type *Mutation* defines the list of mutations that the service enables.
```

Comment:
Suggest either:
The type, *Mutation*, 
or
The *Mutation* type

Reply:
both sound right I vote -> The Mutation type

Reply:
"Mutation type" seems to be more appropriate. Thanks!